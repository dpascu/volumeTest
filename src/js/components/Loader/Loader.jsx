import React from 'react';

/**
 * Loader gif
 */
const Loader = () => (
  <img alt="loading" src="../../../img/loader.gif" />
);

export default Loader;
