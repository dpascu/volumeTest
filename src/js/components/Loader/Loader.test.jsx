/* global describe it expect */
import React from 'react';
import renderer from 'react-test-renderer';

import Loader from './Loader';

describe('Component Loader', () => {
  it('should render component', () => {
    const component = renderer.create(
      <Loader />
    );

    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
