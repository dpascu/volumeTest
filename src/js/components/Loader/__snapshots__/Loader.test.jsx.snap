// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component Loader should render component 1`] = `
<img
  alt="loading"
  src="../../../img/loader.gif"
/>
`;
